CIP-?/README.md

```diff
@@ -0,0 +1,154 @@
+# CIP-?: NFT and Token Authentication
```

Comment:
```suggestion
```

Reply:
yes @emir-olgun - as already suggested in https://github.com/cardano-foundation/CIPs/pull/917#discussion_r1775454310, it's a requirement that this H1 be removed which in any case is redundant with the CIP title in the file metadata.